### ISSUE TITLE
Page freezes when no elements are inside .glide__slides

### ISSUE THREAD

**Issue by thenomadicmann** [glidejs/glide#538]
The page freezes and sends my browser's CPU usage to 100% when I mount an empty slider.

`<div id="home-products__slider" class="glide">

	<div class="glide__track" data-glide-el="track">
	    
	    <div class="glide__slides">

	    </div>

	</div>

</div>`

`var productsSlider = new Glide('#home-products__slider', {
	type: 'carousel',
	perView: 4,
	focusAt: 1,
	gap: 0,
	breakpoints: {
		1199: {
			perView: 3
		},
		991: {
			perView: 3
		},
		767: {
			perView: 2
		},
		575: {
			perView: 1,
			focusAt: 0
		}
	}
});
productsSlider.mount();`

**Comment by thenomadicmann** [comment]
@jedrzejchalubek Any chance of a fix or update to this awesome slider any time soon? We all appreciate your work on this. Just knowing it is still an active project would be great!

**Comment by jedrzejchalubek** [comment]
@thenomadicmann No, any time soon 😞 I surely not going to abandon the project! but lately don't have much time for open source work, sorry about that!
